resources/views/layouts/master.blade.php

```diff
@@ -89,5 +94,6 @@
 <script src="/public/js/flat-ui.min.js"></script>
 <script src="/public/js/donations.js"></script>
 
+<script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-504627a727520762"></script>
```

Comment:
I have currently added my account. If you create an account for this OR we can then move this to the end. For the time being, this should suffice.